app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -328,16 +328,18 @@ def destroy
     desc "Remove content from a repository"
     param :id, :number, :required => true, :desc => "repository ID"
     param 'ids', Array, :required => true, :desc => "Array of content ids to remove"
+    param :content_type, String, :required => false, :desc => N_("srpm, drpm, rpm (default)")
```

Comment:
Can you change this to pull the list from the RepositoryTypeManager.uploadable_content_types ?

Reply:
sorry, should be something like:  RepositoryTypeManager.removable_content_types